frappe/desk/tags.py

```diff
@@ -47,6 +47,11 @@ def remove_tag(tag, dt, dn):
 	"removes tag from the record"
 	DocTags(dt).remove(dn, tag)
 
+@frappe.whitelist()
+def get_tagged_docs(doctype, tag):
```

Comment:
add `frappe.has_permission(doctype, throw=True)` since this is a whitelisted method.